cmrtlib/linux/CMakeLists.txt

```diff
@@ -90,8 +77,20 @@ endif()
 set_target_properties( igfxcmrt PROPERTIES 
   FOLDER CM_RT 
   COMPILE_DEFINITIONS "${CMRT_DEFINES}" 
-  OUTPUT_NAME "igfxcmrt${TARGET_MODIFIER}"
   )
-set_target_properties( igfxcmrt PROPERTIES PREFIX "")
+
+# set the version
+get_cmrt_versions(CMRT_VERSION_MAJOR CMRT_VERSION_MINOR CMRT_VERSION_PATCH)
+set(CMRT_VERSION ${CMRT_VERSION_MAJOR}.${CMRT_VERSION_MINOR}.${CMRT_VERSION_PATCH})
+
+set_target_properties(igfxcmrt PROPERTIES VERSION ${CMRT_VERSION})
+set_target_properties(igfxcmrt PROPERTIES SOVERSION ${CMRT_VERSION_MAJOR})
 target_link_libraries( igfxcmrt dl va rt ${GCC_SECURE_LINK_FLAGS})
 
+if(NOT DEFINED CMAKE_INSTALL_LIBDIR)
+    include(GNUInstallDirs)
+endif()
+
```

Comment:
`include(GNUinstallDirs)` should be used without conditional on the existence of CMAKE_INSTALL_LIBDIR

Reply:
updated